qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -40,8 +40,11 @@ class MgrModuleTest(MgrModuleTestCase):
 
     def test_list_disabled_module(self):
         self._ceph_cmd(['mgr', 'module', 'disable', 'iostat'])
-        self.wait_until_rest_api_accessible()
-        data = self._get('/api/mgr/module')
+        try:
+            data = self._get('/api/mgr/module')
+        except requests.ConnectionError:
+            self.wait_until_rest_api_accessible()
+            data = self._get('/api/mgr/module')
```

Comment:
Perhaps we should extend the `_get()` method to support built-in retries:
```suggestion
        data = self._get(
            '/api/mgr/module',
            tries=2,
            retry_on_exception=requests.ConnectionError,
            wait_func=self.wait_until_rest_api_accessible)
```

Reply:
done, thank you

Reply:
I had to use the lambda wrapper here because without it the `@retry()` decorator of the `wait_until_rest_api_accessible` won't get called inside the other function. Hope its fine. This passed the tests locally for me. will wait for the jenkins

Reply:
Weird... that's I guess because that's part of the same class and decorators on class methods are instantiated after the class is fully defined.